registry.json

```diff
@@ -2598,5 +2598,34 @@
       "rpc": "https://public-node-api.klaytnapi.com/v1/cypress",
       "documentation": "https://docs.klaytn.foundation"
     }
+  },
+  {
+    "id": "meter",
+    "name": "Meter",
+    "coinId": 18000,
```

Comment:
I think `coinId` should be `1000082`

Reply:
@Milerius This one is slip44 id: https://github.com/satoshilabs/slips/blob/master/slip-0044.md, chain id is a fallback if no slip44 id, so it's ok to use 18000